### ISSUE TITLE
Ternary Operators don't generate properly

### ISSUE THREAD

**Issue by pielover928** [ItsDeltin/Overwatch-Script-To-Workshop#145]
```
struct Example{
    define isActive = false;
    public define IsActive: isActive?true:false;
}

globalvar Example e;

rule: "check e"
if(e.IsActive){

}
```

should generate the condition
`(Value In Array(Global Variable(_objectVariable_0), Global Variable(test)) ? True : False) == True;`
but instead generates the condition
`Value In Array(Global Variable(_objectVariable_0), Global Variable(test)) ? True : False == True`
which is not valid in the workshop.

**Comment by pielover928** [comment]
Note the lack of parentheses

**Comment by ItsDeltin** [comment]
Fixed in [v1.5](https://github.com/ItsDeltin/Overwatch-Script-To-Workshop/releases/tag/v1.5).
